### PR TITLE
Update on gsg

### PR DESCRIPTION
**Pull request by srinarayan-srikanthan**
updated gsg based on the comments.

### CODE REVIEW

File: getting-started/README.md [hunk 1]
```diff
@@ -1,21 +1,20 @@
...
 
 ## Understanding OPEA's Core Components
 
 Before moving forward, it's important to familiarize yourself with two key elements of OPEA: GenAIComps and GenAIExamples.
 
 - GenAIComps is a collection of microservice components that form a service-based toolkit. This includes a variety of services such as llm (large language models), embedding, and reranking, among others.
-- While GenAIComps offers a range of microservices, GenAIExamples provides practical, deployable solutions to help users implement these services effectively. Examples include ChatQnA and DocSum, which leverage the microservices for specific applications.
+- GenAIExamples provides practical, deployable solutions to help users implement these services effectively. Examples include ChatQnA and DocSum, which leverage the microservices for specific applications.
```
Comment (arun-gupta) [inline getting-started/README.md:9]: `practical and deployable` instead of `practical, deployable`?

**Review by arun-gupta — CHANGES_REQUESTED** [review]
A minor change, looks good otherwise

**Review by tomlenth — APPROVED** [review]
looks good!

**Review by mkbhanda — APPROVED** [review]
lgtm

**Pull request merged**